vendor.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
```

Comment:
This file should probably not live in the top-level directory, but I didn't have a good idea where to put it.

Reply:
`dist_tools` ? Is there more we can move out of the main directory?

Reply:
well, the `Makefile`, but moving it would require more work than worth it IMHO

Reply:
This is just a build script right? I am ok with it being top level unless we can drop it in a scripts or build type directory.

Reply:
Yepp, it's just used for the "mangle apypie code to make it loadable as a module util of Ansible" step, which we only have to execute when we pull in a new apypie version.